.buildkite/pipeline.yml

```diff
@@ -30,6 +30,13 @@ steps:
   #################
   # Lint
   #################
+  - label: ":rust: Lint"
```

Comment:
@iangmaia Do you think I'd make sence one day to add support for `clippy` & `rustfmt` to our dedicated `linter` agent, instead of running those on the `default` agent (e.g. to benefit from repo caching and all)?

Probably not really a high priority, but figured I'd mention it, also because it might benefit `wordpress-rs` too in the long run I guess 🤔